buildrelease.sh

```diff
@@ -48,13 +48,31 @@ done
 cd docs
 ./builddocs.sh root $projects
 
-echo "Release build complete for the following projects:"
+cd ..
```

Comment:
It might be simpler to change lines 48/49 to:

```sh
(cd docs && ./builddocs.sh root $projects)
```

Then we don't need to come back to this directory.

Or better yes, maybe builddocs.sh should handle being invoked from the root directory... I can look at that later.

Reply:
Done